aiohttp/web_fileresponse.py

```diff
@@ -37,6 +38,13 @@
 
 NOSENDFILE: Final[bool] = bool(os.environ.get("AIOHTTP_NOSENDFILE"))
 
+# File extensions for IANA encodings that will be checked in the order defined.
+# Brotli is specified as a literal until compression is supported.
+ENCODING_EXTENSION: Final[dict[ContentCoding | Literal["br"], str]] = {
```

Comment:
```suggestion
ENCODING_EXTENSION: Final[Dict[str, str]] = {
```

We still need to use legacy typing for older python